modules/ROOT/pages/general-configuration-guide/upload-images.adoc

```diff
@@ -6,7 +6,7 @@
 
 {productname} uploads edited images with the image uploader. This complements {productname}'s image editing functionality.
 
-Local images that are added through other means are also uploaded using this function, such as images added by drag and drop when using the link:{baseurl}/plugins/opensource/paste/#paste_data_images[`paste_data_images`] configuration property, or using the {companyname} link:{baseurl}/plugins/premium/powerpaste/[*PowerPaste* plugin].
+Local images that are added through other means are also uploaded using this function, such as images added by drag and drop when using the xref:plugins/opensource/paste.adoc#paste_data_images[`paste_data_images`] configuration property, or using the {companyname} xref:plugins/premium/powerpaste.adoc[*PowerPaste* plugin].
```

Comment:
The `paste_data_images` xref doesn't resolve as that id doesn't exist. It looks like the id is missing in the `paste_data_images.adoc` file.

- [ ] Fix broken xref

Reply:
Done! I added the missing id in menu.adoc